pkg/kubectl/cmd/namespace.go

```diff
@@ -1,61 +0,0 @@
-/*
-Copyright 2014 Google Inc. All rights reserved.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package cmd
-
-import (
-	"io"
-
-	"fmt"
-
-	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
-	"github.com/spf13/cobra"
-)
-
-func NewCmdNamespace(out io.Writer) *cobra.Command {
```

Comment:
It might be good to have a stub command that prints out an error and says "use X" instead.  We can remove that later.